simulation/avatar/avatar_wrapper.py

```diff
@@ -60,6 +60,8 @@ def set_code(self, code):
         self.code = code
         try:
             exec(code)
+            if Avatar is None:
+                raise Exception('You need to define a class "Avatar"')
```

Comment:
Should this be a UserCodeException? And possibly have text of 'You need to define the class "Avatar"' or 'You need to define a class called "Avatar"'